Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle.yml

```
@@ -20,11 +20,69 @@
+  parent: LPPBaseCartridgeMinigun
   components:
   - type: CartridgeAmmo
     proto: BulletMinigun
     deleteOnSpawn: true
+
+- type: entity
+  id: LPPCartridgeMinigunrubber
+  name: патрон (.10 винтовочный резиновый)
+  parent: LPPBaseCartridgeMinigun
+  components:
+  - type: CartridgeAmmo
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)